packages/kit-bg/src/vaults/impls/dot/Vault.ts

```
@@ -1,14 +1,15 @@
 import { hexToNumber } from '@polkadot/util';
 import {
   checkAddress,
   decodeAddress,
   encodeAddress,
 } from '@polkadot/util-crypto';
-import { decode, methods } from '@substrate/txwrapper-polkadot';
+import { decode } from '@substrate/txwrapper-polkadot';
 import BigNumber from 'bignumber.js';
-import { isEmpty, isNil, isObject } from 'lodash';
+import { md5 } from 'js-md5';
+import { isEmpty, isNaN, isNil, isObject, omit, orderBy } from 'lodash';
```

Biome (2.1.2)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)